modules/ibm-z-secure-execution.adoc

```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
 ----
 gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --verbose --armor --encrypt /path/to/config.ign
 ----
+
+. Follow the fast-track installation procedure to install nodes using the {ibm-name} Secure Execution QCOW image.
 +
 [NOTE]
 ====
-Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` when mounting the Ignition file.
+Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter.
 ====
-+
-When Ignition runs on the first boot, and the decryption is successful, you will see an output like the following example:
+
+.Verification
+
+When you have completed the fast-track installation procedure and Ignition runs on the first boot, verify if decryption is successful.
```

Comment:
```suggestion
* Verify if decryption was successful after the ignition was executed at the first boot.
```

Reply:
Will keep mention of fast-track installation. People tried verification before doing that.